_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -47,212 +40,29 @@ const UpgradeNoticeContent = moduleSettingsForm(
-					toggled: ( false === newPhotonStatus ) ? 'off' : 'on',
-					from: 'warm_welcome'
-				} );
-			}
-
-			// Track any potential Photon CDN toggle switch.
-			if ( this.props.getOptionValue( 'photon-cdn' ) !== newAssetCdnStatus ) {
-				analytics.tracks.recordEvent( 'jetpack_wpa_module_toggle', {
-					module: 'photon-cdn',
-					toggled: ( false === newAssetCdnStatus ) ? 'off' : 'on',
-					from: 'warm_welcome'
-				} );
-			}
-		};
-
 		renderInnerContent() {
-			const foundPhoton = this.props.isModuleFound( 'photon' );
-			const foundAssetCdn = this.props.isModuleFound( 'photon-cdn' );
-
-			// Check if any of the CDN options are on.
-			const siteAcceleratorStatus = this.props.getOptionValue( 'photon' ) || this.props.getOptionValue( 'photon-cdn' );
-
-			// Is at least one of the 2 modules available (not hidden via a module override)?
-			const photonStatus = this.props.getModuleOverride( 'photon' );
-			const assetCdnStatus = this.props.getModuleOverride( 'photon-cdn' );
-			const canDisplaySiteAcceleratorSettings = ( foundPhoton && foundAssetCdn ) && ( 'inactive' !== photonStatus || 'inactive' !== assetCdnStatus );
-
-			// Monitor any changes that should cause our main toggle to appear toggling.
-			let togglingSiteAccelerator;
-			// First Photon activating.
-			if ( ! this.props.getOptionValue( 'photon' ) && this.props.isSavingAnyOption( 'photon' ) ) {
-				if ( this.props.getOptionValue( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			// Then Asset CDN activating.
-			} else if ( ! this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-				if ( this.props.getOptionValue( 'photon' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			// Then Photon deactivating.
-			} else if ( this.props.getOptionValue( 'photon' ) && this.props.isSavingAnyOption( 'photon' ) ) {
-				if ( this.props.getOptionValue( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-
-				// Is the Asset CDN being disabled as well?
-				if ( this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = true;
-				}
-			// Then Asset CDN deactivating.
-			} else if ( this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-				if ( this.props.getOptionValue( 'photon' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			} else {
-				togglingSiteAccelerator = false;
-			}
-
 			return (
 				<div className="jp-upgrade-notice__content">
 					<p>
-						{ __( 'This release of Jetpack brings with it a major new free feature!' ) }
+						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }<br />
+						{ __( 'A new editor? Yes! Learn more.' ) }
```

Comment:
Should there be a link on the "Learn more" text here?

Reply:
I've opted to have the sentences "A new editor? Yes! Learn more." removed completely. I'm not too sure where that link would link to. Let's just remove it.

Reply:
> I'm not too sure where that link would link to. Let's just remove it.

+1

Reply:
Actually let's just link this to the Jetpack.com announcement post. I've got the permanent url when ready.

Reply:
I added the link in https://github.com/Automattic/jetpack/pull/10672/commits/6b5795fe4c16da4312204ee5e341c6659e4f84dd

Reply:
I made some additional changes in d0a4374348fd1d4932194161fd714e38ac8434fe to fix the link syntax and the link URL.